### DISCUSSION TITLE
How to create the database

### DISCUSSION THREAD

**Discussion by mushuthecat** [shekohex/coemu#57] [Q&A]
Hello,

Thanks for your contrib for the coemu project.

Is there any documentation about how to administrate thé server when its running ? ( ban or unban users, offer db stuff or something like that)
for instance how to create custom event or some interaction we can have with the server ?

morever, im not able to process to the last step where can i find the coemu.db in your git ?
regards

**Comment by shekohex** [comment]
Most of these features are not implemented yet.

For the database, you just run `sqlx migrate run --database-url 'sqlite://data/coemu.db?mode=rwc'` and it will create it for you.

**Comment by mushuthecat** [comment]
Hello,

to make easier to deploy your solution i decided to integrate them into a dockerfile:
```

# Utilisez l'image Rust officielle qui inclut Rust et Cargo
FROM rust:latest

# Installez les dépendances nécessaires pour PostgreSQL
RUN apt-get update && apt-get install -y libpq-dev

# Clonez le référentiel GitHub et configurez l'environnement
RUN git clone https://github.com/shekohex/coemu && cd coemu && cp .env.example .env

# Installez le CLI SQLx avec différentes options
RUN cargo install sqlx-cli --no-default-features --features native-tls,postgres
RUN cargo install sqlx-cli --features openssl-vendored
RUN cargo install sqlx-cli --no-default-features --features rustls
RUN cargo install sqlx-cli --no-default-features --features rustls,sqlite
RUN sqlx migrate run --database-url 'sqlite://data/coemu.db?mode=rwc'
RUN cargo auth
RUN cargo game
# Exécutez un script ou configurez votre application si nécessaire
CMD tail -f /dev/null

```

im locked with your last step
sqlx migrate run --database-url 'sqlite://data/coemu.db?mode=rwc'

where is this file ? XD

 > [ 8/10] RUN sqlx migrate run --database-url 'sqlite://data/coemu.db?mode=rwc':
0.275 error: while resolving migrations: No such file or directory (os error 2)

Regards

**Reply by Mejo33** [reply]
run `sqlx migrate run --database-url sqlite://data/coemu.db?mode=rwc`

without ''
